--- old/CMakeLists.txt	2025-02-02 21:56:11.991696365 +0000
+++ new/CMakeLists.txt	2025-02-02 21:58:18.669329187 +0000
@@ -320,7 +320,7 @@
       set(GLEW_SOURCES src/glview/glew-utils.cc)
     endif()

-    find_package(OpenGL REQUIRED QUIET)
+    find_package(OpenGL QUIET)
     target_link_libraries(OpenSCAD PRIVATE ${OPENGL_LIBRARIES})
     message(STATUS "OpenGL: ${OPENGL_LIBRARIES}")
   endif()
